### PR TITLE
Initial Skeleton Commit

### PR DESCRIPTION
**Pull request by carter-greis**
Initial commit for the skeleton files of the project

### CODE REVIEW

File: bin/main.cpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#include <SFML/Graphics.hpp>
+
+int main(int argc, char** argv)
+{
+  // create main window
+  sf::RenderWindow App(sf::VideoMode(800,600,32), "Hello World - SFML");
```
Comment (jkula25) [inline bin/main.cpp]: Name window "Match Made in Hell"

**Comment by jkula25** [comment]
Add a "res" folder for the resources used within the game such as sprites, audio, etc.

File: bin/main.cpp (hunk 1 above)
Comment (skoolaidl) [reply bin/main.cpp]: Go ahead and resolve this @jkula25

**Comment by skoolaidl** [comment]
Looks good to me I am ready to merge once everything is resolved

**Review by kaichenz — APPROVED** [review]

**Pull request merged**